static/js/components/PhotometryValidation.jsx

```diff
@@ -0,0 +1,364 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { useSelector, useDispatch } from "react-redux";
+import Paper from "@mui/material/Paper";
+import { withStyles, makeStyles } from "@mui/styles";
+import { useForm, Controller } from "react-hook-form";
+
+import Dialog from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import MuiDialogTitle from "@mui/material/DialogTitle";
+import IconButton from "@mui/material/IconButton";
+import EditIcon from "@mui/icons-material/Edit";
+import Close from "@mui/icons-material/Close";
+import TextField from "@mui/material/TextField";
+import Typography from "@mui/material/Typography";
+import grey from "@mui/material/colors/grey";
+import Autocomplete, { createFilterOptions } from "@mui/material/Autocomplete";
+
+import Button from "./Button";
+
+import * as PhotometryValidationAction from "../ducks/photometry_validation";
+
+const filter = createFilterOptions();
+
+const defaultExplanationsHighlight = ["GOOD SUBTRACTION"];
+
+const defaultExplanationsReject = ["BAD SUBTRACTION", "COSMIC RAY"];
+
+const defaultExplanations = defaultExplanationsHighlight.concat(
+  defaultExplanationsReject,
+);
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    width: "100%",
+    overflow: "scroll",
+  },
+  eventTags: {
+    marginLeft: "0.5rem",
+    "& > div": {
+      margin: "0.25rem",
+      color: "white",
+      background: theme.palette.primary.main,
+    },
+  },
+  buttons: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  button: {
+    maxWidth: "1.2rem",
+  },
+  buttonIcon: {
+    maxWidth: "1.2rem",
+  },
+}));
+const dialogTitleStyles = (theme) => ({
+  root: {
+    margin: 0,
+    padding: theme.spacing(2),
+  },
+  title: {
+    marginRight: theme.spacing(2),
+  },
+  closeButton: {
+    position: "absolute",
+    right: theme.spacing(1),
+    top: theme.spacing(1),
+    color: grey[500],
```

Comment:
Visible in dark and light modes?

Reply:
@stefanv we don't have a dark mode at all to be honest hahaha, something we hoped some students would work on before, but we don't support dark mode